rg_instructor_analytics/views.py

```diff
@@ -491,6 +495,121 @@ def process(self, request, **kwargs):
         return JsonResponse(data={'student_info': student_info, 'exam_names': exam_names})
 
 
+class CohortView(AccessMixin, View):
+    """
+    Api for cohort statistic.
+    """
+
+    @staticmethod
+    def generate_cohort_by_mean_and_dispersion(student_info):
+        """
+        Generate cohort.
```

Comment:
Please parameter's description in the doc strings

Reply:
done